raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -128,6 +125,8 @@
 public class KafkaRaftClient implements RaftClient {
     private final static int RETRY_BACKOFF_BASE_MS = 100;
 
+    private final static int MAX_RECORDS_DRAINED_PER_ITERATION = 10;
```

Comment:
nit: maybe should be MAX_APPENDS so it is not confused as a maximum number of records.